[ADD] stock_delivery_advance:new_module

jcadhoc opened the pull request.

jcadhoc committed:
[ADD] stock_delivery_advance:new_module

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/stock/pull/808.png)](https://mergebot.adhoc.ar/ingadhoc/stock/pull/808)

jjscarafia commented on stock_delivery_advance/README.rst:
```diff
@@ -0,0 +1,111 @@
+   
+   * Require number of packages before validation
+   * Automatic calculation based on package levels
+   * Validation to prevent zero packages on outgoing transfers
```
borrar este item, es igual al primero de esta lista (Salvo que lo entendimos mal)

jjscarafia committed:
[IMP] stock_ux: add number of packages improvements

Traemos algunas cosas que eran muy basicas desde stock_Delivery_advance, en próximo commit renombramos dicho módulo

jjscarafia committed:
[REF] rename stock_delivery_advance to stock_declared_value

We also remove some functionality that where introduced on stock_ux

jjscarafia committed:
[IMP] stock_declared_value: better ux
